test/cmd/legacy-script.sh

```diff
@@ -324,7 +320,7 @@ runTests() {
 
   # token defined in hack/testdata/auth-tokens.csv
   kube_flags_with_token=(
-    -s "https://127.0.0.1:${SECURE_API_PORT}" --token=admin-token --insecure-skip-tls-verify=true
+    -s "https://127.0.0.1:${SECURE_API_PORT}" "--token=admin-token" "--insecure-skip-tls-verify=true"
```

Comment:
perhaps -s should be quoted too? also if it's not to be expanded we should prefer single quotes:
```suggestion
    '-s' "https://127.0.0.1:${SECURE_API_PORT}" '--token=admin-token' '--insecure-skip-tls-verify=true'
```

Reply:
Same here